lib/GenomeFileUtil/core/GenomeInterface.py

```diff
@@ -299,19 +315,19 @@ def _update_genome(self, genome):
         # NOTE: New Taxon changes should most likely fit in here.
         # NOTE: Metagenome object does not have a 'taxon_ref' field
         if 'taxon_ref' not in genome:
-            genome['taxonomy'], genome['taxon_ref'], domain, genetic_code = self.retrieve_taxon(
-                self.taxon_wsname, genome['scientific_name'])
-            if 'genetic_code' in genome and genome['genetic_code'] != genetic_code:
```

Comment:
Here and below is my best guess at fixing this "update_genome" logic to mirror what it was doing before. I'm not sure that this stuff makes sense at a higher level.

Reply:
I don't understand the overall codebase well enough to provide any advice here, I'm afraid